static/shared/js/emoji.js

```diff
@@ -174,6 +179,42 @@ export function update_emojis(realm_emojis) {
     build_emoji_data(active_realm_emojis);
 }
 
+// This function will provide required parameters that would
+// need by template to render an emoji.
+export function get_emoji_details_by_name(emoji_name) {
+    // // To call this function you must pass an emoji name.
+    if (!emoji_name) {
+        return {};
+    }
```

Comment:
I'd make the parameter required (i.e. do `throw new error` if not provided); when we migrate this file to TypeScript, we'll certainly want to make it a type error to not pass a string here.